recipes-nodes/config-parser/config_parser.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/sh
+
+# Define the allowed keys
+ALLOWED_KEYS="jwtSecret builderSecretKey builderTxSigningKey BuilderBeaconEndpoints"
+
+# Check if jq is available
+if ! command -v jq >/dev/null 2>&1; then
+    echo "Error: jq is not installed" >&2
+    exit 1
+fi
+
+# Check if a file is provided
+if [ $# -ne 1 ]; then
+    echo "Usage: $0 <json_file>" >&2
+    exit 1
+fi
+
+JSON_FILE="$1"
+
+# Check if the file exists
+if [ ! -f "$JSON_FILE" ]; then
+    echo "Error: File not found: $JSON_FILE" >&2
+    exit 1
+fi
+
+# Function to validate hex string
+validate_hex() {
+    case "$1" in
+        0x[0-9a-fA-F]*) return 0 ;;
+        *) return 1 ;;
+    esac
+}
+
+# Function to validate URL
+validate_url() {
+    case "$1" in
+        http://*|https://*) return 0 ;;
```

Comment:
I don't trust the wildcard expression here, let's mark it as review required

Reply:
I agree, we can add more restrictions or checks for it.
Currently, that function is only used once for the BuilderBeaconEndpoints variable to make sure that it starts with a valid URL and not setting arbitrary data in there.
But sure, we can synthesis the logic even more